lib/ui/view/threads/all_threads/threads_viewmodel.dart

```diff
@@ -12,18 +12,17 @@ class ThreadsViewModel extends BaseViewModel {
   final _navigationService = locator<NavigationService>();
 
   List<UserPost> userPosts = userPost;
-  
 
   void initialise() {
     //refresh page
+    print('Hello there');
```

Comment:
Please use a logger and ensure your code is clean by removing all print statement with commented codes